traefik-crds/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: traefik-crds
+description: A Traefik based Kubernetes ingress controller
+type: application
+version: 0.0.1
```

Comment:
```suggestion
version: 3.2.0-rc1
```

Reply:
I don't know which versioning we should use. But traefik one, doesn't seem to be a good idea as we aren't installing traefik only CRDs.